src/app/admin/ui/painting/add-painting/add-painting.component.html

```diff
@@ -0,0 +1,96 @@
+<div class="container">
+  <form enctype="multipart/form-data" class="col-md-6 my-5 mx-auto" #addPaintingForm="ngForm" (ngSubmit)="mySubmit(addPaintingForm)">
+    <h2 class="h1 ml-5 text-center">Add New Painting</h2>
+    <div class="form-group row">
+      <label class="col-2 col-form-label">Name</label>
+      <div class="col-10">
+        <input class="form-control" type="text" name="name" ngModel placeholder="Type Painting Name">
+      </div>
+    </div>
+     <div class="form-group row">
+      <label class="col-2 col-form-label">Dimension</label>
+      <div class="col-10">
+        <input class="form-control" type="text" name="deminsions" ngModel placeholder="Type The Dimension">
+      </div>
+    </div>
+    <div class="form-group row">
+     <label class="col-2 col-form-label">Date</label>
+     <div class="col-10">
+       <input class="form-control" type="date" name="addingDate" ngModel placeholder="Type The Date">
+     </div>
+   </div>
+    <div class="form-group row">
+      <label class="col-2 col-form-label">C-Type</label>
+      <div class="col-10">
+        <input class="form-control" type="text" name="colorsType" ngModel placeholder="Type The Colors Type">
+      </div>
+    </div>
+    <div class="form-group row">
+       <label class="col-2 col-form-label">Price</label>
+       <div class="col-10">
+         <input class="form-control" type="number" name="price" ngModel placeholder="Type The Price">
```

Comment:
I am expecting to have everything as component , like price component ...
no complex html